scripts/sfpshow

```diff
@@ -587,6 +589,23 @@ def eeprom(port, dump_dom, namespace):
     sfp.get_eeprom()
     sfp.display_eeprom()
 
+# 'eeprom' subcommand
```

Comment:
i thought this "info" does not come under as subcommand of 'eeprom' since we will remove 'eeprom' command in future?

Reply:
Yes, this is a typo in comment. I have corrected it now.